pkg/kubelet/kubelet.go

```diff
@@ -1052,7 +1052,10 @@ func (kl *Kubelet) podFieldSelectorRuntimeValue(fs *api.ObjectFieldSelector, pod
 	if err != nil {
 		return "", err
 	}
-
+	switch internalFieldPath {
+	case "status.podIP":
```

Comment:
Ok

On Aug 12, 2015, at 6:45 PM, Brian Grant <notifications@github.com> wrote:

In pkg/kubelet/kubelet.go
<https://github.com/kubernetes/kubernetes/pull/12595#discussion_r36923772>:

> @@ -1052,7 +1052,10 @@ func (kl *Kubelet) podFieldSelectorRuntimeValue(fs *api.ObjectFieldSelector, pod
>  	if err != nil {
>  		return "", err
>  	}
> -
> +	switch internalFieldPath {
> +	case "status.podIP":

Is the reason this can't go into ExtractFieldPathAsString because that
function is the same for all resource types? A comment here to that effect
would be useful.

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/12595/files#r36923772>.